src/form/FormAPI/CustomForm.php

```
@@ -0,0 +1,173 @@
+class CustomForm extends IForm {
+
+    /** @var array<int|string> */
+    private array $labelMap = [];
+
+    /** @var array<callable> */
+    private array $validationMethods = [];
+
+    /**
+     * @param callable(?array): void|null $callable
+     */
+    public function __construct(?callable $callable) {
```

PHPStan (2.1.15)

Method pocketmine\form\FormAPI\CustomForm::__construct() has parameter $callable with no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)